README.md

```diff
@@ -9,6 +9,7 @@ List sorted by CFP end date descending (oldest and probably already finished las
 | 2018.11.01 | 2018.11.27-28 | Paris, France | [Flowcon](http://flowcon.fr/) | [here](https://www.papercall.io/flowcon) | Agile, Kanban, Software craftsmanship, ... |
 | 2018.09.30 | 2019.02.04-06 | Stockholm, Sweden | [Jfokus](http://www.jfokus.se/) | [here](http://www.jfokus.se/jfokus/proposals.jsp) | JVM, NoSQL, Mobile... |
 | 2018.09.30 | 2018.11.15 | London, England | [Redis Day London](https://www.eventbrite.com/e/redis-day-london-tickets-47448914982) | [here](https://www.papercall.io/redis-day-london-2018) | Redis performance, tuning, production lessons learned, ... |
+| 2018.09.21 | 2018.10.23 | Paris, France | [MDF 2018](http://meilleurdevdefrance.com/) | [here](hhttps://www.papercall.io/mdf2018) | Day to day, experience feedback, basics... |
```

Comment:
You've got a typo in your link, friend. 👋

Reply:
indeed, thanks :)